Source/Core/DolphinQt/Config/Graphics/AdvancedWidget.cpp

```diff
@@ -34,7 +34,9 @@ AdvancedWidget::AdvancedWidget(GraphicsWindow* parent)
 
   connect(parent, &GraphicsWindow::BackendChanged, this, &AdvancedWidget::OnBackendChanged);
   connect(&Settings::Instance(), &Settings::EmulationStateChanged, this,
-          [=](Core::State state) { OnEmulationStateChanged(state != Core::State::Uninitialized); });
+          [=, this](Core::State state) {
```

Comment:
Nothing else seems to be captured by the `=`, it was only ever capturing `this`

You should get rid of the `=, `

The same appears to apply for most (if not all) of the following changes.